lib/active_fulfillment/services/shopify_api.rb

```diff
@@ -68,7 +68,9 @@ def fetch_tracking_data(order_numbers, options = {})
     private
 
     def request_uri(action, data)
-      URI.parse "#{@callback_url}/#{action}.#{@format}?#{data.to_query}"
+      url = URI.parse "#{@callback_url}/#{action}.#{@format}?#{data.to_query}"
+      url.path.squeeze!('/')
```

Comment:
Um, I wouldn't go so far as to squeeze all the path. 
How about 
```
url = URI.join(@callback_url, "#{action}.#{@format}?#{data.to_query}")
```

This automatically takes care of the double `/` between host and path.